src/Native/Runtime/PalRedhawk.h

```diff
@@ -656,6 +742,9 @@ inline UInt8 * PalNtCurrentTeb()
 #define OFFSETOF__TEB__ThreadLocalStoragePointer 0x2c
 #elif defined(_AMD64_)
 #define OFFSETOF__TEB__ThreadLocalStoragePointer 0x58
+#elif defined(_ARM64_)
+// FIXME:ARM64
```

Comment:
I like what you have now. I just like to be able to search the codebase for `@TODO`.